css/style.css

```diff
@@ -0,0 +1,56 @@
+/**{
+    margin: 0;
+    padding:0;
+    box-sizing: border-box;
+    background-color: beige;
+  }
+  */
+  #navbar {
+    position: fixed;
+    min-width: 200px;
+    top: 0px;
+    left: 0px;
+    width: 300px;
+    height: 100%;
+    border-right: solid;
+    border-color: black;
+  }
+  body{
+    margin-left: 320px;
+    background-color: bisque;
+    font-family: arial;
+    min-width: 200px;
+  }
+  #navbar{
+    text-align:center;
+  }
+
+  section header{
+    font-size: larger;
+    font-weight: bolder;
+  }
+#Main_Tittle{
+    text-align: center;
+  }
+@media only screen and (max-width: 800px) {
+    #main-doc {
+      margin-left: -10px;
+    }
+    #navbar{
+        font-size: small;
+        color:aqua;
+        position: absolute;
+        height: 200px;
+        width: 100%;
+    }
+    body {
```

Comment:
You could use a class attribute for multiples elements:

- [ ] If you want to create a CSS rule for multiple elements, use a class attribute instead.